src/Akka.Persistence.MongoDb.Hosting.Tests/MongoDbJournalOptionsSpec.cs

```diff
@@ -0,0 +1,153 @@
+
+            baseConfig.GetString("akka.persistence.journal.plugin").Should().Be("akka.persistence.journal.custom");
+
+            var config = baseConfig.GetConfig("akka.persistence.journal.custom");
+            config.Should().NotBeNull();
+            config.GetString("connection-string").Should().Be(options.ConnectionString);
+            config.GetBoolean("auto-initialize").Should().Be(options.AutoInitialize);
+            config.GetString("collection").Should().Be(options.Collection);
+            config.GetString("metadata-collection").Should().Be(options.MetadataCollection);
+            config.GetBoolean("use-write-transaction").Should().Be(options.UseWriteTransaction);
+            config.GetBoolean("legacy-serialization").Should().Be(options.LegacySerialization);
+            config.GetTimeSpan("call-timeout").Should().Be(options.CallTimeout);
+        }
+
+        const string Json = @"
+        {
+          ""Logging"": {
+            ""LogLevel"": {
+              ""Default"": ""Information"",
+              ""Microsoft.AspNetCore"": ""Warning""
+            }
+          },
+          ""Akka"": {
+            ""JournalOptions"": {
+              ""ConnectionString"": ""mongodb://localhost:27017"",
+              ""UseWriteTransaction"": ""true"",
+              ""Identifier"": ""custommongodb"",
+              ""AutoInitialize"": true,
+              ""IsDefaultPlugin"": false,
+              
+              ""Collection"": ""CustomEnventJournalCollection"",
+              ""MetadataCollection"": ""CustomMetadataCollection"",
+              ""LegacySerialization"" : ""true"",
+              ""CallTimeout"": ""00:10:00"",
+              ""Serializer"": ""hyperion"",
+            }
+          }
+        }";
+
+        [Fact(DisplayName = "MongoDbJournalOptions should be bindable to IConfiguration")]
+        public void JournalOptionsIConfigurationBindingTest()
+        {
+            var stream = new MemoryStream(Encoding.UTF8.GetBytes(Json));
+            var jsonConfig = new ConfigurationBuilder().AddJsonStream(stream).Build();
+
+            var options = jsonConfig.GetSection("Akka:JournalOptions").Get<MongoDbJournalOptions>();
+            options.ConnectionString.Should().Be("mongodb://localhost:27017");
+            options.UseWriteTransaction.Should().BeTrue();
+            options.Identifier.Should().Be("custommongodb");
+            options.AutoInitialize.Should().BeTrue();
+            options.IsDefaultPlugin.Should().BeFalse();
+            options.Collection.Should().Be("CustomEnventJournalCollection");
+            options.MetadataCollection.Should().Be("CustomMetadataCollection");
+            options.LegacySerialization.Should().BeTrue();
+            options.CallTimeout.Should().Be(10.Minutes());
+            options.Serializer.Should().Be("hyperion");
+
+            // Dispose called here as project not using latest language features.
+            stream.Dispose();
```

Comment:
Added this to both tests as the using declarations are not currently available in the language features for this project and it caused build failures. Do you have a preferred way of handling this?